Doc.podspec

```diff
@@ -0,0 +1,42 @@
+Pod::Spec.new do |s|
```

Comment:
Why does Jazzy need a separate PodSpec?

It seems it will be difficult to maintain this because we have to keep two podspecs in sync. Is there any way to keep the config in just one podspec?